pages/events/[id].tsx

```diff
@@ -0,0 +1,89 @@
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useGetEventByIdQuery } from 'generated/graphql';
+import { Typography } from 'components/atoms/Typography/Typography';
+import { useState } from 'react';
+
+import { EventDetails } from 'components/templates/events/eventDetails';
+import { BudgetDetails } from 'components/templates/events/budgetDetails';
+
+const EventDetailsPage = () => {
+	const router = useRouter();
+	const id = router.query.id as string;
+	const { data, loading } = useGetEventByIdQuery({ variables: { id } });
+	const [isEvent, setIsEvent] = useState('event');
```

Comment:
What I meant was something like that
```ts
type View = 'event' | 'budget'
const [view, setView] = useState<View>('event');
```

Then we can simply add a new tab

Reply:
My fault will be added soon